src/frontend/ai-reviewer-admin/src/App.scss

```diff
@@ -9,4 +9,16 @@ body,
 }
 .content {
   flex: 1 0 auto;
+  flex-grow: 1;
+  margin-top: 100px;
+}
+
+.header-div {
```

Comment:
The header and footer already has their own class.  Do you think it makes to use **.bcgov-header, .bcgov-footer** instead?

Reply:
Do you mean to just rename this class to one of those? This is just to force the header and footer to render over the drawer. Maybe I'm not understanding what you mean

Reply:
No, I mean replace this line 16 with
`.bcgov-header, .bcgov-footer {`

And remove the <div> wrapper around the footer.
Just a little cleaner.

Reply:
Oh I see what you mean now. Ya that sounds good:)